config/templates/chef-marketplace-cookbooks/non_root_solo.rb.erb

```diff
@@ -0,0 +1,8 @@
+cookbook_path '/opt/chef-marketplace/embedded/cookbooks'
+cache_path '/tmp/chef-marketplace/local-mode-cache'
```

Comment:
Hrm, we could do that.  We don't actually care about any of these files at all which is why I used /tmp.  Perhaps I should just wipe that directory entirely after the chef run completes?